tests/bits/step-10-high-order.cc

```diff
@@ -180,10 +178,10 @@ compute_pi_by_perimeter()
 int
 main()
 {
+  initlog();
+  deallog << std::setprecision(16);
   deallog << std::setprecision(16);
-  logfile << std::setprecision(16);
```

Comment:
and you don't set the precision of the logfile any more (as you do for some other tests).